estate/models/estate_property_tags.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields,models;
```

Comment:
we don't have semi-colon(`;`) in python
```suggestion
from odoo import fields, models
```